File: LLM-Translations/GPT-4/codenet/Python/Java/s592421258.java
import java.util.Scanner;

public class s592421258 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String doubledInput = input + input;
        String target = scanner.nextLine();
        System.out.println(doubledInput.contains(target) ? "Yes" : "No");
    }
}

// 

File: LLM-Translations/GPT-4/codenet/Python/Go/s592421258.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var input string
	fmt.Scan(&input)
	doubleInput := input * 2
	var check string
	fmt.Scan(&check)

	if strings.Contains(doubleInput, check) {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}
// 
